torch/distributed/rpc/api.py

```diff
@@ -370,6 +370,13 @@ def remote(to, func, args=None, kwargs=None):
         need to explicitly copy GPU tensors to CPU before using them as
         arguments or return values of ``func``.
 
+    .. warning ::
+        The ``remote`` API does not copy storages of argument tensors until
+        sending them over the wire, which could be done by a different thread
+        depending on the RPC backend type. The caller should make sure that the
+        contents of those tensors stay intact until the returned RRef is
+        confirmed by the owner.
```

Comment:
The user has no way currently to know when the RRef has been confirmed by the owner right? Should we mention that either the user should never modify the passed in Tensors or only modify them after calling RRef.to_here()? This does add some restrictions to the API, but I'm not sure if there is a better way.

Reply:
[`RRef.confirmed_by_owner()`](https://pytorch.org/docs/master/rpc.html#torch.distributed.rpc.RRef.confirmed_by_owner) is a public API. Let me add a link to that API here.

Reply:
Aaah didn't know about that! Sounds good to me!